daemon/logger/journald/journald.go

```diff
@@ -36,6 +36,26 @@ func init() {
 	}
 }
 
+// sanitizeKeyMode returns the sanitized string so that it could be used in journald.
+// In journald log, there are special requirements for fields.
+// Fields must be composed of uppercase letters, numbers, and underscores, but must
+// not start with an underscore.
```

Comment:
Should we have a test on that (the fact that a string starting with an underscore is handled somehow 👼).